src/witan/app/util.clj

```diff
@@ -34,6 +34,15 @@
                params)))
       true)))
 
+(defn is-a-number? [txt]
+  (try (number? (read-string txt))
+       (catch NumberFormatException _ false)))
+
+(defn to-uuid
```

Comment:
sounds like we don't know what things might be convertable to UUID? Would  a multi method be better (as in more extensible).

```
(defmulti to-uuid type)
(defmethod to-uuid java.lang.String [id] ...)
(defmethod to-uuid :default [id] id)
```